pkg/kapp/diff/change_set_test.go

```diff
@@ -266,3 +266,60 @@ metadata:
 
 	require.Equal(t, expectedDiff, actualDiff, "Expected diff to match")
 }
+
+func TestChangeSet_RebaseWithNilNew_And_UnexpectedChanges(t *testing.T) {
```

Comment:
We generally stick to Pascal Case for test names. How does `TestRebaseWithNilNewAndUnexpectedChanges` sound?

Reply:
I can definitely change that. I just copied the format from the other tests above, so I'm not opinionated one way or another.  :smile: